Makefile

```diff
@@ -18,11 +18,14 @@ RUNC_LINK := $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 export GOPATH := $(CURDIR)/Godeps/_workspace
 
 MAN_DIR := $(CURDIR)/man/man8
-MAN_PAGES = $(shell ls $(MAN_DIR)/*.8)
+MAN_PAGES = $(shell ls $(MAN_DIR)/*.8.gz)
 MAN_PAGES_BASE = $(notdir $(MAN_PAGES))
 MAN_INSTALL_PATH := ${PREFIX}/share/man/man8/
 
 VERSION := ${shell cat ./VERSION}
+RPM_VERSION := $(shell echo $(VERSION) | sed "s/-[^-]*$$//g")
```

Comment:
Can we use single quotes for these regular expressions -- `$$` makes me think of "get the PID of the shell" which is super-confusing as this is a makefile.